.commitlintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["@commitlint/config-conventional"]
```

Comment:
what be this

Reply:
I think this is the config file used by husky for the git hooks

Reply:
should it be in the PR tho? looks a new thing

Reply:
Yeah, I double checked and this is correct. We had a `commitlint.config.cjs` and because I transitioned it to a JSON file, the file name is a different.

Source: https://github.com/conventional-changelog/commitlint#config